airbyte-ci/connectors/pipelines/pipelines/helpers/git.py

```diff
@@ -74,9 +72,30 @@ async def get_modified_files_in_commit(current_git_branch: str, current_git_revi
         return await get_modified_files_in_commit_remote(current_git_branch, current_git_revision)
 
 
+def get_modified_files_since_local(current_git_revision: str, since: str) -> Set[str]:
+    airbyte_repo = git.Repo()
+    diffed_git_revision = airbyte_repo.git.log(current_git_revision, f"--before='{since}'", "--max-count=1", "--pretty=%H").strip()
+    return get_modified_files_local(current_git_revision, diffed_git_revision)
+
+
+async def get_modified_files_since_remote(current_git_branch: str, current_git_revision: str, since: str) -> Set[str]:
```

Comment:
I don't find these two helpers being used? Or am I missing coffee? If they're not used yet I'm not sure the version bump should happen in this branch.

Reply:
You're missing coffee. In any case I just got rid of them. I'm going to merge this stack just to fix the stacked pull request behaviour.